db/seeds.rb

```diff
@@ -1,7 +1,5 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+Template.create(
+  :title => 'Basic',
+  :description => 'The very basic approval workflow that supports multi-level approver groups through email notification',
```

Comment:
Suggest changing the start to "A basic approval ...".

If it was "very basic" it would not include multi-level approver groups.  😄